ports/protobuf/vcpkg.json

```
@@ -1,7 +1,6 @@
 {
   "name": "protobuf",
-  "version-string": "3.14.0",
-  "port-version": 1,
+  "version-string": "3.15.3",
```

```suggestion
  "version-semver": "3.15.3",
```
For the package version, please this doc https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md